spec/views/idv/in_person/verify_info/show.html.erb_spec.rb

```diff
@@ -0,0 +1,198 @@
+require 'rails_helper'
+
+RSpec.describe 'idv/in_person/verify_info/show.html.erb' do
+  let(:address_pii) do
+    {
+      address1: Faker::Address.street_address,
+      address2: Faker::Address.secondary_address,
+      city: Faker::Address.city,
+      state: Faker::Address.state_abbr,
+      zipcode: Faker::Address.zip,
+    }
+  end
+
+  let(:ssn_pii) do
+    {
+      ssn: '666' + Faker::Number.number(digits: 6).to_s,
```

Comment:
Nice use of a prefix and `Faker` to ensure the ssn will be recognized as fake. 👍🏻